CDCgov/NEDSS-Modernization
# Disable Compare Investigations button based on permissions

## Description

Disable/Enable Compare Investigations button based on permissions

## Tickets

* [[Jira Ticket](url)](https://cdc-nbs.atlassian.net/browse/CNFT1-1499)

## Steps to verify

 Open the NBS test env url in safari browseR: 

Enter user name as state and click on submit button

on the main header section on Home Screen, click on the open investigations link

Click on any of the patient name that is [Dupont, Ishmael](https://app.test-cdc-nbs.eqsandbox.com/nbs/LoadNavbar.do?ContextAction=GlobalInvestigations&initLoad=true#)

click on events tab

select the checkboxes beside 2 investigations and observe that compare investigations button should be disabled now if user doesn't have permissions.
